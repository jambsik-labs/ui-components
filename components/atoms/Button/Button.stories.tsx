import React from 'react';
import { Button } from './Button';

export default {
    title: 'atoms/Button',
    component: Button,
};

export const defaultCase = () => (
    <Button disabled={false} variant="primary">
        Beep
    </Button>
);
