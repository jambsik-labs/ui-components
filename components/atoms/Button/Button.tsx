/** @jsxImportSource theme-ui */

import React from 'react';
import { ButtonProps } from '@theme-ui/components';
import { Button as ThemeUIButton } from 'theme-ui';

const buttonStyles = {
    cursor: 'pointer',
};

export const Button = (props: ButtonProps) => {
    return <ThemeUIButton sx={buttonStyles} {...props} />;
};
