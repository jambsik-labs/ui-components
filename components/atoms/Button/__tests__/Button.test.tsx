import React from 'react';
import { render } from '@testing-library/react';
import { Button } from '../Button';

describe('Button test', () => {
    it('Should render component without errors', () => {
        const { container } = render(<Button />);

        expect(container).toMatchSnapshot();
    });
});
