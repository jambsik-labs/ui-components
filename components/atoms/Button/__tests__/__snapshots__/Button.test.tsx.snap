// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test Should render component without errors 1`] = `
<div>
  <button
    class="css-1hyo902-Button"
  />
</div>
`;
